src/components/ADDPage.tsx

```diff
@@ -20,6 +20,8 @@ const ADDPage = observer(() => {
     navigation.navigate('Excel');
   };
 
+  getLanguageStore.setAsyncData();
```

Comment:
this shouldnt be here, either useEffect, or in the function when we are adding.

Reply:
done